<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1111</width>
    <height>730</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:white
</string>
  </property>
  <widget class="QLabel" name="labelPulseIcon">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>120</y>
     <width>81</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color : transparent;
background-color:transparent;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../../../../../Downloads/free-icon-love-14415504.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="labelFrame">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>1051</width>
     <height>311</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:#F5C26B;
border-radius:50px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelDog">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>180</y>
     <width>591</width>
     <height>551</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:transparent</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../../../../../Downloads/dog (1).png</pixmap>
   </property>
  </widget>
  <widget class="QPushButton" name="btnFeed">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>540</y>
     <width>191</width>
     <height>151</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-style:outset;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../../../../../Downloads/free-icon-pet-food-6903249.png</normaloff>../../../../../Downloads/free-icon-pet-food-6903249.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>150</width>
     <height>150</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="btnHeart">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>240</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-style:outset;
background-color:transparent;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../../../../../Downloads/heart.png</normaloff>../../../../../Downloads/heart.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>60</width>
     <height>60</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="labelFoodEmpty">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>590</y>
     <width>131</width>
     <height>131</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../../../../../Downloads/pet-bowl.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCameraPage">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>400</y>
     <width>101</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:white;
border-radius:50px;
border-color:#4169E1;
border-width:10px;
border-style:solid;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelFeedInfo">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>220</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:transparent;</string>
   </property>
   <property name="text">
    <string>마지막 배식시간 : </string>
   </property>
  </widget>
  <widget class="QLabel" name="labelPulse">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>140</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Noto Sans Mono</family>
     <pointsize>29</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:transparent;</string>
   </property>
   <property name="text">
    <string>89</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelFeedTime">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>220</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:transparent;</string>
   </property>
   <property name="text">
    <string>12 : 20</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelActInfo">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>260</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:transparent;</string>
   </property>
   <property name="text">
    <string>오늘의 활동량</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelActivity">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>260</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:transparent;</string>
   </property>
   <property name="text">
    <string>활발함</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>100</y>
     <width>251</width>
     <height>141</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:40px;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="labelNameInfo">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>30</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>DejaVu Sans Mono</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>이름</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelAgeInfo">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>80</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>DejaVu Sans Mono</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-style:outset;</string>
    </property>
    <property name="text">
     <string>나이</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelPawIcon">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../../../../Downloads/paws.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="labelPawIcon2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../../../../Downloads/paws.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="labelName">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>40</y>
      <width>71</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>이상범</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelAge">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>90</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>24</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="labelTempBody">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>140</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Noto Sans Mono</family>
     <pointsize>29</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:transparent;</string>
   </property>
   <property name="text">
    <string>38</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelTempC">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>140</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Noto Sans Mono</family>
     <pointsize>29</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:transparent;</string>
   </property>
   <property name="text">
    <string>°C</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelTempInfo">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>100</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:transparent;</string>
   </property>
   <property name="text">
    <string>체온</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelPulseInfo">
   <property name="geometry">
    <rect>
     <x>940</x>
     <y>100</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:transparent;</string>
   </property>
   <property name="text">
    <string>심박수</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelWater">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>600</y>
     <width>131</width>
     <height>111</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../../../../../Downloads/water.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="labelFood">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>560</y>
     <width>131</width>
     <height>121</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../../../../../Downloads/free-icon-pet-food-1950702.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSetting">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>81</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-style:outset;
background-color:transparent;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../../../../../Downloads/preferences.png</normaloff>../../../../../Downloads/preferences.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="btnGpsPage">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>400</y>
     <width>101</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:white;
border-radius:50px;
border-color:#4169E1;
border-width:10px;
border-style:solid;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnPlayPage">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>400</y>
     <width>101</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:white;
border-radius:50px;
border-color:#4169E1;
border-width:10px;
border-style:solid;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelDateTime">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>10</y>
     <width>181</width>
     <height>18</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:transparent;
</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <zorder>labelFrame</zorder>
  <zorder>labelDog</zorder>
  <zorder>labelPulseIcon</zorder>
  <zorder>btnFeed</zorder>
  <zorder>btnHeart</zorder>
  <zorder>labelFoodEmpty</zorder>
  <zorder>btnCameraPage</zorder>
  <zorder>labelFeedInfo</zorder>
  <zorder>labelPulse</zorder>
  <zorder>labelFeedTime</zorder>
  <zorder>labelActInfo</zorder>
  <zorder>labelActivity</zorder>
  <zorder>frame</zorder>
  <zorder>labelTempBody</zorder>
  <zorder>labelTempC</zorder>
  <zorder>labelTempInfo</zorder>
  <zorder>labelPulseInfo</zorder>
  <zorder>labelWater</zorder>
  <zorder>labelFood</zorder>
  <zorder>btnSetting</zorder>
  <zorder>btnGpsPage</zorder>
  <zorder>btnPlayPage</zorder>
  <zorder>labelDateTime</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
